idd/Energy+.idd.in

```diff
@@ -60003,23 +60003,23 @@ Coil:WaterHeating:AirToWaterHeatPump:VariableSpeed,
        \note PLR = part load ratio (heating delivered/steady state heating capacity).
        \note Use curve coefficients of 1,0,0 or leave this field blank when neglecting performance impacts
        \note due to variations in part load ratio.
-  N12, \field Rated Water Heating Capacity at Speed 1
+  N12, \field Speed 1 Rated Water Heating Capacity
        \required-field
        \type real
        \units W
        \minimum> 0
        \note Heating capacity at the rated inlet air temperatures, rated condenser inlet
        \note water temperature, rated air flow rate, and rated water flow rate.
        \note Can optionally include condenser pump heat.
-  N13, \field Rated Water Heating COP at Speed 1
+  N13, \field Speed 1 Rated Water Heating COP
        \type real
        \units W/W
        \minimum> 0
        \default 3.2
        \note Heating coefficient of performance at the rated inlet air and water temperatures,
        \note rated condenser inlet water temperature, rated air flow rate, and rated water flow rate.
        \note Can optionally include condenser pump power and evaporator fan power (see fields below).
-  N14, \field Rated Sensible Heat Ratio at Speed 1
+  N14, \field Speed 1 Rated Sensible Heat Ratio
```

Comment:
Fixes inconsistent field names for speeds 1 to 3 for VS Water Heating AtoWHP object